app/Repositories/UserRepository.php

```diff
@@ -32,7 +32,7 @@ public function create($data)
         $user = new User();
         $user->name = $data->getName();
         $user->email = $data->getEmail();
-        $user->twitter = $data->getTwitter();
+        $user->twitter = str_replace('@', '', $data->getTwitter());
```

Comment:
I find the following line more semantic and slighly more readable:

```
$user->twitter = ltrim($data->getTwitter(), '@');
```

Reply:
Good call!  Making that change now...